### DISCUSSION TITLE
Need to be internationalized

### DISCUSSION THREAD

**Discussion by sphlls-hohhot** [grafana/grafana#43586] [Plugins]
Need some international language switching: such as Chinese, Japanese, etc., it is too painful for customers to use the above

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
